components/opponent/commons/opponent.lua

```diff
@@ -251,12 +252,15 @@ function Opponent.resolve(opponent, date, options)
 		for _, player in ipairs(opponent.players) do
 			if options.syncPlayer then
 				PlayerExt.syncPlayer(player, {savePageVar = not Opponent.playerIsTbd(player)})
+				if not player.team then
+					player.team = PlayerExt.syncTeam(player.pageName:gsub(' ', '_'), nil)
+				end
+				if player.team then
+					player.team = TeamTemplate.resolve(player.team, date)
+				end
```

Comment:
Why is this only done on syncPlayer? Is that intended?

Reply:
Yeah, the current way is syncTeam gets enabled when **prizePool:setConfigDefault('syncPlayers', true)** (syncPlayers is where flags get retrieved on its own) existed inside the PPT custom. So for wikis that dont want this feature doesnt have it by default

Reply:
Yes, but this is TT.resolve? Why was it moved into scope of syncPlayer?

Reply:
I think can be simply because I messed up and put it there (when it shouldnt be)